ModularTegustation/tegu_items/associations/machines.dm

```diff
@@ -213,8 +213,8 @@
 	resistance_flags = INDESTRUCTIBLE
 
 /obj/structure/timelock/Initialize()
-	..()
-	addtimer(CALLBACK(src, PROC_REF(die)), 15 MINUTES)
+	addtimer(CALLBACK(src, PROC_REF(die)), 5 MINUTES) // NOW!!
```

Comment:
The 15 minutes is actually intentional to force players to prepare, and give them time to get registered at Hana instead of just rushing the backstreets.

It stops people from immediately rushing in with minimal gear while some people are still getting set up

Reply:
fair enuff